source/ceylon/buffer/codec/IncrementalCodec.ceylon

```diff
@@ -23,8 +22,10 @@ shared class ChunkConvert<ToMutable, FromImmutable, ToSingle, FromSingle>(conver
     shared Boolean done => remainder.empty;
```

Comment:
Rename it to something else maybe, but it's essential that the user of `ChunkConvert` know if the codec is satisfied.